public/locales/fr/translation.json

```
@@ -0,0 +1,79 @@
+{
+    "seed_classification_interface": "Interface de Classification des semences",
+    "developed_by_ai_lab": "Développé par le Laboratoire d'IA",
+    "sign_in": "SE CONNECTER",
+    "email_address": "Adresse email",
+    "password": "Mot de passe",
+    "remember_me": "Se souvenir de moi",
+    "dont_have_account": "Vous n'avez pas de compte? Inscrivez-vous",
+    "forgot_password": "Mot de passe oublié?",
+    "capture_btn": "CAPTURER",
+    "switch": "CHANGER DE CAMÉRA",
+    "batch": "IMPORTER PAR LOT",
+    "load": "CHARGER",
+    "save": "ENREGISTRER",
+    "model_selection": "SÉLECTIONNER LE MODÈLE",
+    "classify": "CLASSIFIER",
+    "annotate": "ANNOTER",
+    "directories": "RÉPERTOIRES",
+    "captures": "CAPTURES",
+    "results": "RÉSULTATS | ",
+    "video_feed": "FLUX VIDÉO",
+    "capture_feed": "CAPTURE",
+    "please_select_a_class": "Veuillez sélectionner une classe",
+    "please_enter_a_seed_count": "Veuillez entrer un nombre de semences",
+    "please_enter_a_zoom_level": "Veuillez entrer un niveau de zoom",
+    "please_select_an_image": "Veuillez sélectionner une image",
+    "no_file_selected": "Aucun fichier sélectionné",
+    "invalid_file_type": "Type de fichier invalide",
+    "batch_upload_images": "Importation par lot d'images",
+    "folder_name": "Nom du dossier",
+    "class": "Classe",
+    "seed_count": "Nombre de semences",
+    "zoom_level": "Niveau de zoom",
+    "select_files": "Sélectionner des fichiers",
+    "transfer_status": "Statut du transfert",
+    "upload": "Importer",
+    "create_new_directory": "Créer un nouveau répertoire",
+    "directory_name": "Nom du répertoire",
+    "create": "Créer",
+    "cancel": "Annuler",
+    "choose_media_device": "Choisir un périphérique multimédia",
+    "load_image": "Charger une image",
+    "classification_model_selection": "Sélection du modèle de classification",
+    "done": "Terminé",
+    "model_selection_popup": "Sélection du modèle:",
+    "save_capture": "Enregistrer la capture",
+    "capture_name": "Nom de la capture",
+    "delete_directory": "Supprimer le répertoire",
+    "are_you_sure_you_want_to_delete_": "Êtes-vous sûr de vouloir supprimer ",
+    "delete": "Supprimer",
+    "close": "Fermer",
+    "top_results": "Meilleurs résultats",
+    "feedback": "Rétroaction",
+    "submit": "Soumettre",
+    "bounding_box": "Cadre d'objet",
```

```suggestion
    "bounding_box": "Rectangle englobant",
```
https://vitrinelinguistique.oqlf.gouv.qc.ca/fiche-gdt/fiche/8400239/rectangle-englobant